templates/corporate/upgrade.html

```diff
@@ -106,7 +111,7 @@ <h1>$<span class="due-today-price"></span></h1>
                         <input type="hidden" name="licenses" id="automatic_license_count" value="{{ seat_count }}" />
                     </div>
                     {% else %}
-                    <div id="license-manual-section" class="input-box upgrade-page-field">
+                    <div id="license-manual-section" class="input-box upgrade-page-field license-management-section">
                         <p class="not-editable-realm-field">
```

Comment:
| before | after |
| --- | --- |
| <img width="808" alt="Screenshot 2023-11-28 at 12 14 20 PM" src="https://github.com/zulip/zulip/assets/25124304/2c87ee58-6f71-4116-b1f0-60f26d888504"> | <img width="808" alt="Screenshot 2023-11-28 at 12 10 PM" src="https://github.com/zulip/zulip/assets/25124304/7a869803-7a51-4106-aa57-c05c221cf9e2"> |

Reply:
Cool, that looks better to me.